terraform/addons/mdm/outputs.tf

```diff
@@ -1,8 +1,6 @@
 output "extra_environment_variables" {
   value = merge(var.enable_apple_mdm == false ? {} : {
     FLEET_MDM_APPLE_SERVER_ADDRESS = var.public_domain_name
-    }, var.enable_windows_mdm == false ? {} : {
-    FLEET_DEV_MDM_ENABLED = "1"
```

Comment:
For infra folks reviewing this - I removed the feature flag environment variable but left everything else intact in the terraform, this means the `var.enable_windows_mdm` variable is still there (as I think it should, since it is used to define the WSTEP cert or not). TBD if the default value should now be set to `true` as it is for Apple?

Reply:
Thanks @rfairburn , in order to keep the impact of this PR as focused as possible I'll leave it as it is for now.